src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.test.js

```diff
@@ -57,6 +57,10 @@ jest.mock('../../queries', () => ({
     sourceFiles: [],
     isLoading: false,
   }),
+  useMarcSpecifications: jest.fn().mockReturnValue({
```

Comment:
Probably this can be removed because there's no hook like this

Reply:
Yeah - forget to remove - thanks